src/components/MegadraftEditor.js

```diff
@@ -387,10 +387,39 @@ export default class MegadraftEditor extends Component {
   render() {
     const hideSidebarOnBlur = this.props.hideSidebarOnBlur || false;
     const i18n = this.props.i18n[this.props.language];
+    let classEditor = "megadraft-editor";
+    let contentState = this.props.editorState.getCurrentContent();
+
+    // If the user changes block type before entering any text, we can
+    // either style the placeholder or hide it.
+    // Class with styling to spacing placeholder.
+    if (!contentState.hasText()) {
+      switch (
+        contentState
+          .getBlockMap()
+          .first()
+          .getType()
+      ) {
+        case "ordered-list-item":
+        case "unordered-list-item":
+          classEditor += " placeholder-list";
+          break;
+        case "header-two":
+          classEditor += " placeholder-header-two";
+          break;
+        case "blockquote":
+          classEditor += " placeholder-blockquote";
+          break;
+        default:
+          classEditor += "";
+          break;
+      }
+    }
```

Comment:
@lucasferreiralimax, what do you think about create a method to abstract this logic?

Reply:
I created the function for manipulate and to abstract logic also adjustment an class with an identifier with template string.

See the work @JeanPinzon